gubernator/github/handlers.py

```diff
@@ -29,13 +29,20 @@
 
 import classifier
 import models
+from .. import secrets
```

Comment:
Please do not use relative imports: https://google.github.io/styleguide/pyguide.html?showone=Packages#Packages

Reply:
This is the only reasonable way to make this work with pylint. `import secrets` works otherwise. I could make pylint ignore it, but that's worse?

Reply:
Appengine sites don't have full package names that make avoiding relative imports possible.

Reply:
https://github.com/bazelbuild/rules_appengine? :smiling_imp: 

At least add a `# erick-lint: disable relative import because reasons` type note on the import lint

Reply:
done